integration_test/gce/gce_testing.go

```diff
@@ -415,14 +415,15 @@ func lookupTrace(ctx context.Context, logger *log.Logger, vm *VM, window time.Du
 // time series. An error is returned if the evaluation fails.
 // A return value of (nil, nil) indicates that the evaluation succeeded
 // but returned no data.
-func nonEmptySeries(logger *log.Logger, it *monitoring.TimeSeriesIterator) (*monitoringpb.TimeSeries, error) {
+func nonEmptySeries(logger *log.Logger, it *monitoring.TimeSeriesIterator) ([]*monitoringpb.TimeSeries, error) {
 	// Loop through the iterator, looking for at least one non-empty time series.
+	tsList := make([]*monitoringpb.TimeSeries, 0)
 	for {
 		series, err := it.Next()
 		logger.Printf("nonEmptySeries() iterator supplied err %v and series %v", err, series)
 		if err == iterator.Done {
-			// Either there were no data series in the iterator or all of them were empty.
-			return nil, nil
+			// Success, we found a time series with len(series.Points) > 0.
```

Comment:
What if the iterator is empty? It is awkward for a function named `nonEmptySeries` to produce an empty series without an error.